src/modules/fancyzones/editor/FancyZonesEditor/Utils/WorkAreaData.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows;
+using System.Windows.Forms;
+
+namespace FancyZonesEditor.Utils
+{
+    public class WorkAreaData
+    {
+        public string Id { get; private set; }
+
+        public Rect WorkAreaRect { get; private set; }
+
+        public Rect WorkAreaRectScaled { get; private set; }
+
+        public Screen WorkAreaScreen { get; private set; }
+
+        public int Dpi { get; private set; }
+
+        public WorkAreaData(string id, int dpi, int x, int y)
+        {
+            Id = id;
+            Dpi = dpi;
+
+            Screen[] screens = Screen.AllScreens;
+            foreach (Screen screen in screens)
+            {
+                var screenX = ConvertDpi(screen.WorkingArea.X);
+                var screenY = ConvertDpi(screen.WorkingArea.Y);
```

Comment:
From C++ we send the screen coordinates that are not scaled, and in C# we also get the coordinates that are not scaled.
Is there a scenario where we need to use `ConvertDpi`?